hack/make/dependencies.make

```diff
@@ -29,7 +29,7 @@ CONTROLLER_TOOLS_VERSION ?= v0.10.0
 KIND_VERSION ?= v0.17.0
 HELM_VERSION ?= v3.10.0
 YQ_VERSION ?= v4.30.8
-ISTIOVERSION ?= 1.17.0
+ISTIOVERSION ?= 1.20.0-rc.0
```

Comment:
so question for me is how do we ensure we bump this once released? I don't want to hold open the PR if we don't need to  but also don't want to forget @laurafitzgerald perhaps we could add an issue and bump this and move to top of backlog so we always see it?

Reply:
Tracking that here https://github.com/Kuadrant/multicluster-gateway-controller/issues/665